### PR TITLE
(ENG-891) Add baseMealSlug and proteinAddOn tags to recipe data retrieval

### PR DESCRIPTION
**Pull request by thunguy**
## Description
The focus of this PR is to add two more recipe-level category tag values, `baseMealSlug` and `proteinAddOn`, to the return value of `get_formatted_recipes_data()` for label printing purposes. This PR also ensures that `baseMealSlug`, `proteinAddOn`, and all other existing recipe-level category tags are identified via their id--`categoryId`. Automated tests have been updated to confirm the expected behavior(s).

## Test Plan
Open python interactive within galley-sdk: `$ python`
`>>> from pprint import pprint` (optional: to apply pretty print format to returned data)
`>>> from galley.formatted_queries import *`
 `>>> pprint(get_formatted_recipes_data(['cmVjaXBlOjE4Mzk4NA==']))`

> **Expectation:** A return value **similar** to:
> ```python3
> [{'baseMealSlug': 'persephone-salad',
>   'description': None,
>   'externalName': None,
>   'id': 'cmVjaXBlOjE4Mzk4NA==',
>   'ingredients': [ ... ],
>   'mealContainer': 'ts48'
>   'mealType': 'lunch',
>   'notes': None,
>   'nutrition': { ... },
>   'proteinAddOn': 'garbanzo-beans',
>   'proteinType': 'vegan',
>   'weight': 113.4}]
>```

## Run Automated Unittests
Exit python interactive and stay within the galley-sdk directory to run tests:
`(galley-env) $ python3 -m unittest tests/test_formatted_queries.py`
**[OPTIONAL SANITY CHECK]** `(galley-env) $ python -m unittest tests/test_*.py`

## Versioning
v0.15.1 --> v0.16.0

### CODE REVIEW

File: galley/enums.py [hunk 1]
```diff
@@ -14,15 +15,30 @@ class PreparationEnum(Enum):
     STANDALONE = 'cHJlcGFyYXRpb246MjgzMzQ='
     TWO_OUNCE_RAM = 'cHJlcGFyYXRpb246MjgxMTU='
     THREE_OUNCE_RAM = 'cHJlcGFyYXRpb246MjgxMTQ='
+    PROTEIN_ADDON = 'cHJlcGFyYXRpb246Mjg1NDM='
```
Comment (mollykmcglone) [inline galley/enums.py:18]: I don't think this is necessary?

**Review by dip-thistle — COMMENTED** [review]
Tested locally! The changes to have additional data for recipe working as expected. I can approve once other engineer's comments are addressed!

File: galley/enums.py (hunk 1 above)
Comment (thunguy) [reply galley/enums.py]: Oops! Forgot to remove this following my investigation. Removed.

**Comment by ddhanesha** [comment]
@thunguy please resolve merge conflicts

**Comment by dip-thistle** [comment]
@thunguy please resolve merge conflicts.

**Comment by thunguy** [comment]
@dip-thistle Resolved. 😄

**Review by dip-thistle — APPROVED** [review]
Let’s get this merged!

**Pull request merged**